### PR TITLE
Get top movers from Zerion

### PR DESCRIPTION
**Pull request by jinchung**
Replace top movers source from dapple to Zerion.
Hook up charts and pricing info in expanded state.
Include an interval that will refetch top movers data every 10 minutes.

Screen Shot 2021-01-28 at 10 26 00 PM

Screen Shot 2021-01-29 at 12 48 17 PM

### CODE REVIEW

**Review by osdnk — APPROVED** [review]
Let us merge this quickly!

File: src/components/coin-row/TopMoverCoinRow.js [hunk 1]
```diff
@@ -34,11 +34,13 @@ const PADDING_BETWEEN_ITEMS = 26;
...
 export const measureTopMoverCoinRow = async ({
   change,
-  price,
+  native,
   truncatedName,
 }) => {
   const { width: nameWidth } = await measureTopRowText(truncatedName);
-  const { width: priceWidth } = await measureBottomRowText(price);
+  const { width: priceWidth } = await measureBottomRowText(
+    native?.price?.display || ''
```
Comment (osdnk) [inline src/components/coin-row/TopMoverCoinRow.js:42]:
```suggestion
    native?.price?.display ?? ''
```

File: src/components/discover-sheet/TopMoversSection.js [hunk 2]
```diff
@@ -1,39 +1,19 @@
-import { toLower } from 'lodash';
 import React, { useCallback, useMemo } from 'react';
-import { useSelector } from 'react-redux';
-import { handleSignificantDecimals } from '../../helpers/utilities';
-import {
-  useAccountAssets,
-  useAccountSettings,
-  useTopMovers,
-} from '../../hooks';
+import { useTopMovers } from '../../hooks';
```
Comment (osdnk) [inline src/components/discover-sheet/TopMoversSection.js:2]:
```suggestion
import { useTopMovers } from '@rainbow-me/hooks';
```

**Review by osdnk — COMMENTED** [review]
nits

File: src/components/coin-row/TopMoverCoinRow.js (hunk 1 above)
Comment (brunobar79) [reply src/components/coin-row/TopMoverCoinRow.js:42]: Fancy

**Review by brunobar79 — APPROVED** [review]
🚀

**Review by skylarbarrera — APPROVED** [review]
LGTM

**Pull request merged**